hw2/reverse.js_HW

```diff
@@ -0,0 +1,33 @@
+document.getElementById("reverse").onclick = () => {
+  let num = document.getElementById("input");
+  let out = document.getElementById("output");
+  let newnum = num.value;
+  let num1 = newnum;
+  let count = 0;
+
+  //counting the number of digits provided by the user
+  while (newnum / 10 > 1) {
+    ++count;
```

Comment:
I would probably do the same but the Airbnb style guide [13.6](https://github.com/airbnb/javascript#variables--unary-increment-decrement) suggests avoiding using unary increments. Maybe we should avoid this in the future homework assignments